### PR TITLE
[CFP-155] Replace app css helpers with that of the GOV.UK Design System

### PR DESCRIPTION
**Pull request by naseberry**
#### What

Replace app css helpers with that of the GOV.UK Design System

#### Ticket

[Replace stylesheet helper classes](https://dsdmoj.atlassian.net/browse/CFP-155)

#### Why

To continue the migration away from the deprecated GOV.UK Elements to GOV.UK Frontend (GOV.UK Design System)

#### How

| GOV.UK Elements | GOV.UK Frontend |
| --- | ----------- |
| visually-hidden, visuallyhidden | govuk-visually-hidden, govuk-visually-hidden-focusable |
| clearfix | govuk-clearfix |

### CODE REVIEW

File: app/webpack/stylesheets/_claims_importer.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
   .js-enabled & {
     input[type="file"],
     .no-file-selected [type="submit"] {
-      @include visually-hidden;
+      @include govuk-visually-hidden;
```
Comment (jsugarman) [inline app/webpack/stylesheets/_claims_importer.scss:14]: **yak-shaving-warning** but I do not believe we need this file at all as the import was removed

**Review by jsugarman — APPROVED** [review]
👍  looks fine to me but have not tested UI.

- one file that I am pretty confident can be removed entirely (separate PR perhaps)
- question: is there any functional difference between the old `visually-hidden` and `visuallyhidden`?

File: app/webpack/stylesheets/_claims_importer.scss (hunk 1 above)
Comment (naseberry) [reply app/webpack/stylesheets/_claims_importer.scss:14]: Great, thank you. I will remove this.

**Comment by naseberry** [comment]
> 👍 looks fine to me but have not tested UI.
> 
> * one file that I am pretty confident can be removed entirely (separate PR perhaps)
> * question: is there any functional difference between the old `visually-hidden` and `visuallyhidden`?

- There are a few imported files that can be removed, to be done in a separate PR
- There are no differences between `visually-hidden` and `visuallyhidden`, see: https://github.com/alphagov/govuk_elements/blob/master/assets/sass/elements/_helpers.scss

**Pull request merged**